lib/models/asset-size-printer.js

```diff
@@ -39,16 +38,14 @@ AssetPrinterSize.prototype.makeAssetSizesObject = function () {
   var assets = files
     // Skip test files
     .filter(function (file) {
-      var filename = path.basename(file);
-      return !testFileRegex.test(filename);
+      return !testFileRegex.test(file);
```

Comment:
Yes, this is not necessary. I figured we could remove the `path`, but your argument is compelling. Changed.